arangod/RocksDBEngine/RocksDBEngine.cpp

```diff
@@ -118,10 +118,23 @@
 #include <iomanip>
 #include <limits>
 
+#undef USE_SST_INGESTION
+
 using namespace arangodb;
 using namespace arangodb::application_features;
 using namespace arangodb::options;
 
+#ifdef USE_SST_INGESTION
+namespace {
+void cleanUpExtFiles(std::string_view path) {
```

Comment:
Quick question: what does `Ext` stand for here? It is not obvious to the reader. Should this be "temp" or "temporary"?

Reply:
Stands for external, as, before rocksdb moved them, they would be in an external non-standard directory, but changed name